packages/back-end/types/dimension.d.ts

```diff
@@ -3,6 +3,7 @@ export interface DimensionInterface {
   organization: string;
   owner: string;
   datasource: string;
+  description: string;
```

Comment:
This field is not optional, but none of the existing Mongo records will have a description.  We should either make this optional (so type errors will be found) or add a migration in the Model to force every doc to have a description set.

Same for segments.